test_scripts/Polices/build_options/032_ATF_P_TC_PTU_Restarting_Policy_Table_Exchange_HTTP.lua

```diff
@@ -74,156 +81,187 @@ local function DelayedExp(time)
+            print ("fourth retry time: " .. timeout[5])
+            return true
+          elseif exp.occurences == 6 and timeout[5] ~= (os.time() - thirdTryTime) then
+            fourthTryTime = os.time()
+            print ("Wrong fourth retry time! Expected: " .. timeout[5] .. " Actual: " .. (os.time() - thirdTryTime) )
+            return false
+          end
 
-function Test:TestStep_Register_New_App_And_Check_New_PTU_Starting()
-local CorIdRAI2 = self.mobileSession2:SendRPC("RegisterAppInterface",
-    {
-      syncMsgVersion =
-      {
-        majorVersion = 3,
-        minorVersion = 0
-      },
-      appName = "AnotherAppName",
-      isMediaApplication = true,
-      languageDesired = "EN-US",
-      hmiDisplayLanguageDesired = "EN-US",
-      appID = "7654321",
-      deviceInfo =
+          if exp.occurences == 7 and timeout[6] == (os.time() - fourthTryTime) then
+            print ("fifth retry time: " .. timeout[6])
+            return true
+          elseif exp.occurences == 7 and timeout[6] ~= (os.time() - fourthTryTime) then
+            print ("Wrong fifth retry time! Expected: " .. timeout[6] .. " Actual: " .. (os.time() - fourthTryTime) )
+            return false
+          end
+        end
+
+        return false
+
+      end)
+    :Times(#seconds_between_retries + 2) -- 6 HTTP, 1 LOCK_SCREEN_ICON_URL
+
+    DelayedExp((timeout[1] + timeout[2] + timeout[3] + timeout[4] + timeout[5] + timeout[6])*1000 + 2*60000) --msec
+    EXPECT_RESPONSE(CorIdRAI, { success = true, resultCode = "SUCCESS"})
+  end
+
+  function Test:TestStep_StartSession2()
+    self.mobileSession2 = mobile_session.MobileSession(self, self.mobileConnection)
+    self.mobileSession2:StartService(7)
+  end
+
+  function Test:TestStep_Register_New_App_And_Check_New_PTU_Starting()
+    local CorIdRAI2 = self.mobileSession2:SendRPC("RegisterAppInterface",
       {
-        os = "Android",
-        carrier = "Megafon",
-        firmwareRev = "Name: Linux, Version: 3.4.0-perf",
-        osVersion = "4.4.2",
-        maxNumberRFCOMMPorts = 1
-      }
-    })
-  EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "HTTP"})
-  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
-  self.mobileSession2:ExpectResponse(CorIdRAI2, {success = true, resultCode = "SUCCESS"})
-end
+        syncMsgVersion =
+        {
+          majorVersion = 3,
+          minorVersion = 0
+        },
+        appName = "AnotherAppName",
+        isMediaApplication = true,
+        languageDesired = "EN-US",
+        hmiDisplayLanguageDesired = "EN-US",
+        appID = "7654321",
+        deviceInfo =
+        {
+          os = "Android",
+          carrier = "Megafon",
+          firmwareRev = "Name: Linux, Version: 3.4.0-perf",
+          osVersion = "4.4.2",
+          maxNumberRFCOMMPorts = 1
+        }
+      })
+    EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "LOCK_SCREEN_ICON_URL"}, {requestType = "HTTP"}):Times(2)
```

Comment:
@istoimenova, Since both apps are in NONE HMI level "OnSystemRequest" can come through any of them.
So catching of it has to be corrected.